server/server.go

```diff
@@ -55,7 +57,9 @@ type Config struct {
 
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
+	f.StringVar(&cfg.HTTPListenHost, "server.http-listen-host", "", "HTTP server listen host.")
```

Comment:
Is there some precedent for calling this "host" rather than "address" ?

Reply:
@bboreham Shall I change it in #167?

Reply:
Yes please, unless someone can explain why "host" is the correct word.

Reply:
Done